bindings/node/lib/autoEncrypter.js

```diff
@@ -276,3 +284,47 @@ module.exports = function (modules) {
 
   return { AutoEncrypter };
 };
+
+const decryptedKeys = Symbol.for('@@mdb.decryptedKeys');
```

Comment:
This is probably a non-issue, I haven't used the global registry for symbols much. Is it important that this be registered as early as possible? Maybe it should be moved to be above the `module.exports = function (modules) {` module func that wraps everything in here.

Reply:
I think it doesn’t really matter where it’s defined, I just moved it into `decorateDecryptionResult` itself if that works for you

Reply:
Yea I supposed as long as the string matches it doesn't matter where the first Symbol.for('xx') call is made. Boxing it inside the helper function LGTM